docs/cryptodoc/src/05_08_ml_dsa.rst

```diff
@@ -0,0 +1,443 @@
+.. _pubkey/ml_dsa:
+
+ML-DSA
+======
+
+Botan implements the Module-Lattice-based Digital Signature Algorithm (ML-DSA)
+in :srcref:`src/lib/pubkey/dilithium`. The implementation is based on
+[FIPS-204]_. The list of supported algorithm parametres is listed in the table
+:ref:`pubkey/ml_dsa/params`.
+
+.. _pubkey/ml_dsa/params:
+
+.. table:: Supported ML-DSA Parameter Sets (see Table 1 of [FIPS-204]_)
+
+   +------------+-----------+-----------+---------------+-----------+--------------+-----------------+------------------+------------------+---------------+--------------+---------------+----------------+
+   | Mode       | :math:`q` | :math:`n` | :math:`\zeta` | :math:`d` | :math:`\tau` | :math:`\lambda` | :math:`\gamma_1` | :math:`\gamma_2` | :math:`(k,l)` | :math:`\eta` | :math:`\beta` | :math:`\omega` |
+   +============+===========+===========+===============+===========+==============+=================+==================+==================+===============+==============+===============+================+
```

Comment:
Maybe remove $n$? It does not appear in the table and of FIPS-204 and is also never referenced in this chapter.

Reply:
Yeah, I went back and forth with that one, too. Will remove it.